Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptSearchController.java

```diff
@@ -87,6 +87,8 @@ public String search(HttpServletRequest req, ModelMap model, @RequestParam(defau
         if (results.hasErrors()) {
             return "conceptsearch";
         }
+        String str = conceptSearchBean.getWord().trim();
+        conceptSearchBean.setWord(str);
```

Comment:
isnt' that a different ticket? why is that here?

Reply:
This was a mistake. Instead of creating a branch out of develop, I think I created a branch out of CCP 243 and that is why that change is here and as I addressed one of the review comments and committed the code its here. Sorry about this.

Reply:
please remove and commit to the correct branch

Reply:
removed the change from this branch